src/fortran/urqmd-3.4/sources/init.f

```diff
@@ -342,7 +342,6 @@ subroutine init
 c    &           bdist=nucrad(Ap)+nucrad(At)+2*CTParam(30)
 c hjd1
  215        bimp=sqrt(ranf(0))*bdist
-cdh         if (bimp<bmin) goto 215
             if (bimp .lt.  bmin) goto 215
```

Comment:
wtf?!

Reply:
No suggestion to change anything, just a comment of my surprise to see this kind of code.